custom_components/tahoma/sensor.py

```diff
@@ -34,7 +42,8 @@
         key="core:BatteryState",
         name="Battery",
         device_class=sensor.DEVICE_CLASS_BATTERY,
-        value=lambda value: str(value).capitalize(),
+        unit_of_measurement=PERCENTAGE,
+        value=lambda value: OVERKIZ_BATTERY_TO_HOME_ASSISTANT.get(value, value),
```

Comment:
```suggestion
        value=lambda value: OVERKIZ_BATTERY_TO_HA.get(value, value),
```

Reply:
Why default to value?

Reply:
This was the previous behavior and will catch a situation when Somfy (or another vendor) has a different key in the `core:BatteryState`.